prime-angular-frontend/src/app/core/services/error-handler.service.ts

```diff
@@ -29,16 +32,7 @@ export class ErrorHandlerService implements ErrorHandler {
       }
     } else {
       // Client error has occurred (Angular Error, ReferenceError...)
+      dialogLogger.log(error);
     }
-
-    const message = (error.message)
-      ? error.message
-      : error.toString();
-    const url = router.url;
-
-    logger.error(message, { url });
```

Comment:
Should the error still go to `LoggerService`?  I'm thinking about PRIME-1325 which would record it to web-api.

Reply:
> Should the error still go to `LoggerService`? I'm thinking about PRIME-1325 which would record it to web-api.

Yes, and I believe PRIME-1325 would also log the client error in the else statement